tests/compute/atomics-buffer.slang

```diff
@@ -2,6 +2,13 @@
 
 //TEST(compute):COMPARE_COMPUTE_EX:-slang -compute
 //TEST(compute):COMPARE_COMPUTE_EX:-slang -compute -dx12
```

Comment:
I guess this test proves me wrong about `RWBuffer` and atomics, but I'm honestly very confused by that. I wonder what the restrictions are in D3D to make that workable.